tests/framework/extensions/provisioning/creates.go

```diff
@@ -376,6 +404,20 @@ func CreateProvisioningAirgapCustomCluster(client *rancher.Client, clustersConfi
 		rolesPerNode[nodes.Quantity] = finalRoleCommand
 	}
 
+	if clustersConfig.PSACT == string(provisioninginput.RancherBaseline) {
+		err := clusters.CreateRancherBaselinePSACT(client, clustersConfig.PSACT)
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	if clustersConfig.PSACT == string(provisioninginput.RancherBaseline) {
+		err := clusters.CreateRancherBaselinePSACT(client, clustersConfig.PSACT)
+		if err != nil {
+			return nil, err
+		}
+	}
+
```

Comment:
I think this is a duplication of the above. Also creates 7 lines diff from 2.7 PR.

Reply:
You're right 😅 fixing that now

Reply:
Resolved